src/common/arrow/arrow_array_scan.cpp

```diff
@@ -247,8 +247,8 @@ static void scanArrowArrayStruct(const ArrowSchema* schema, const ArrowArray* ar
 static void scanArrowArrayDenseUnion(const ArrowSchema* schema, const ArrowArray* array,
     ValueVector& outputVector, ArrowNullMaskTree* mask, uint64_t srcOffset, uint64_t dstOffset,
     uint64_t count) {
-    auto types = ((const int8_t*)array->buffers[0]) + srcOffset;
-    auto dstTypes = UnionVector::getTagVector(&outputVector)->getData();
+    auto types = ((const uint8_t*)array->buffers[0]) + srcOffset;
+    auto dstTypes = (uint16_t*)UnionVector::getTagVector(&outputVector)->getData();
```

Comment:
Double check coverage

Reply:
We don't add pytest in codecov that's why it is marked as uncovered